enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/tx/CatchupPollingProcess.java

```diff
@@ -62,7 +65,7 @@
  */
 public class CatchupPollingProcess extends LifecycleAdapter
 {
-    enum Timeouts implements TimeoutName
+    enum Timeouts implements TimerName
```

Comment:
If `TimeoutName` was changed to `TimerName` shouldn't `Timeouts` be renamed to `Timers`?

Reply:
I agree, but didn't care at the time because this was outside the timer-framework. Will fix though!